src/Objs/defaultPageAttributes.ts

```diff
@@ -0,0 +1,17 @@
+export const defaultPageAttributes = {
```

Comment:
Currently this is only used on `Homepage` and `Page`.

I think the following Obj types would also benefit from this:
* `SubnavigationOverview`
* `ProductOverview`, `ProductCategory` and `Product`